tests/core/test_eodata_io.py

```diff
@@ -408,65 +420,23 @@ def test_cleanup_different_compression(fs_loader, eopatch):
     folder = "foo-folder"
     patch_folder = "patch-folder"
     with fs_loader() as temp_fs:
-        temp_fs.makedir(folder)
-
-        save_compressed_task = SaveTask(
-            folder, filesystem=temp_fs, compress_level=9, overwrite_permission="OVERWRITE_FEATURES"
-        )
-        save_noncompressed_task = SaveTask(
-            folder, filesystem=temp_fs, compress_level=0, overwrite_permission="OVERWRITE_FEATURES"
-        )
+        save_task = SaveTask(folder, filesystem=temp_fs, overwrite_permission="OVERWRITE_FEATURES")
         bbox_path = fs.path.join(folder, patch_folder, "bbox.geojson")
-        compressed_bbox_path = bbox_path + ".gz"
-        mask_timeless_path = fs.path.join(folder, patch_folder, "mask_timeless", "mask.npy")
-        compressed_mask_timeless_path = mask_timeless_path + ".gz"
-
-        save_compressed_task(eopatch, eopatch_folder=patch_folder)
-        save_noncompressed_task(eopatch, eopatch_folder=patch_folder)
-        assert temp_fs.exists(bbox_path)
-        assert temp_fs.exists(mask_timeless_path)
-        assert not temp_fs.exists(compressed_bbox_path)
-        assert not temp_fs.exists(compressed_mask_timeless_path)
-
-        save_compressed_task(eopatch, eopatch_folder=patch_folder)
-        assert not temp_fs.exists(bbox_path)
-        assert not temp_fs.exists(mask_timeless_path)
-        assert temp_fs.exists(compressed_bbox_path)
-        assert temp_fs.exists(compressed_mask_timeless_path)
-
-
-def test_cleanup_different_compression_zarr(eopatch):
-    # zarr and moto dont work atm anyway
-    # slightly different than regular one, since zarr does not use gzip to compress itself
-    _skip_when_appropriate(None, True)
-    folder = "foo-folder"
-    patch_folder = "patch-folder"
-    with TempFS() as temp_fs:
-        temp_fs.makedir(folder)
+        timestamps_path = fs.path.join(folder, patch_folder, "timestamps.json")
 
-        save_compressed_task = SaveTask(
-            folder, filesystem=temp_fs, compress_level=9, overwrite_permission="OVERWRITE_FEATURES", use_zarr=True
-        )
-        save_noncompressed_task = SaveTask(
-            folder, filesystem=temp_fs, compress_level=0, overwrite_permission="OVERWRITE_FEATURES", use_zarr=True
-        )
-        bbox_path = fs.path.join(folder, patch_folder, "bbox.geojson")
-        compressed_bbox_path = bbox_path + ".gz"
-        mask_timeless_path = fs.path.join(folder, patch_folder, "mask_timeless", "mask.zarr")
-        wrong_compressed_mask_timeless_path = mask_timeless_path + ".gz"
+        # need to manually save uncompressed feature
```

Comment:
i'd also add this for bbox and timestamps

Reply:
added